src/frontend/src/lib/components/transactions/Transactions.svelte

```diff
@@ -24,16 +24,20 @@
 			token.name === $routeToken && $routeNetwork && token.network.id.description === $routeNetwork
 	);
 
+	let timeout: NodeJS.Timeout | undefined;
```

Comment:
just for consistency

```suggestion
	let timer: NodeJS.Timeout | undefined;
```